Omnibus.md

```diff
@@ -3422,4 +3422,304 @@ This spell prevents Undead from attacking the spellcaster while it is active.
+  background.
+
+Throwing a legal event gives an EH certain privileges and responsibilities
+above and beyond a normal player.
+
+* To validate your event you must confirm it electronically with the official
+  Event List Administrator within one month of the date of the event. Also
+  within that month, you must also send a summary of reports of rules
+  violations and the conclusion of investigations into them, as well as all
+  consequences levied, to the Arbitration Committee if any reports were filed.
+  EHs and the AC have the responsibility to maintain the privacy of those
+  involved in reports to the extent possible.
+* If you release a Realms Magic Item at your event you must submit the name,
+  physical description, and magical description to the Keeper of the List of
+  Realms Magic Items within one month of that event. The actual prop for the
+  item does not need to be distributed to players at that time, but an EH that
+  does not designate a magic item for an event within one month forever loses
+  the opportunity to release an item for the event (see
+  [Section 7.4: Magic Item Rules](#magic-item-rules) below).
+* As an EH you qualify to attend the Event Holders' Council meeting for the
+  year following your event (see
+  [Section 7.5: The Event-Holders' Council](#the-event-holders-council) below).
+
+## 7.3: Arbitration Committee ## {#arbitration-committee}
+
+The Arbitration Committee is a group of five members and two alternates
+nominated and elected yearly at the EHC that seeks to resolve issues by
+speaking or meeting with the affected individuals and, in the case where
+actions are detrimental to the community, levy penalties against individuals.
+The Arbitration Committee can be reached at <arbitration@lists.realmsnet.net>.
+
+When individuals are found responsible for breaking rules by a simple majority
+of the Arbitration Committee, the committee may levy punishment consistent
+with both the charge and a goal of working to resolve the issue in the future.
+
+At an emergency meeting of the EHC, a 2/3 vote of the council can overturn or
+amend the actions of the Arbitration Committee.
+
+The players elected to the 2018 Arbitration Committee are Angela Gray,
+Ben Greene, Hannah Blood, John Berrini, and Keith Cronyn.
+
+The two alternates elected to the 2018 Arbitration Committee are James Murphy
+and Matt Butler.
+
+## 7.4: Magic Item Rules ## {#magic-item-rules}
+
+An EH is permitted to create one non-event specific magic item
+(a “Realms Magic Item”) at each event for which they receive EH credit.
+Only one magic item may be released for each event, even if there are multiple
+EHs. Such magic items will retain their powers and abilities at all subsequent
+events at which they are pre-registered/checked in, and approved. Magic items
+are released on the date of the event the EH is creating them for and must be
+submitted to the Keeper of the List of Realms Magic Items within one month of
```

Comment:
How would you feel about making links to the various people who are elected each year?  AC and OEC have their own sections, making it easy to link to them.  The others are in 7.5.

My thinking is that, if I'm looking at the requirements for what I need to report to whom, I might want a quick link to the contact information for that person.

This would also mean adding links in the list following "Throwing a legal event gives an EH certain privileges and responsibilities above and beyond a normal player.".  In that same list, I'd probably have "Realms Magic Item" link to `#magic-item-rules`.